venues/ICLR.cc/2019/Conference/webfield/authorWebfield.js

```diff
@@ -252,6 +252,10 @@ function renderContent(notes, submittedNotes, assignedNotePairs, assignedNotes,
     if (!_.includes(COMMENT_EXCLUSION, note.invitation)) {
       commentNotes.push(note);
     }
+    if (note.invitation === SUBMISSION_ID &&
+        !_.includes(_.map(authorNotes, n=>n.id), note.id)) {
+      authorNotes.push(note);
```

Comment:
Also, I don't really understand what this does. It seems like you're adding a submitted note to the author notes list, but only if it already exists in the author note list??

Reply:
Sorry, my bad. I missed the !, so you only add it if it's not in the list.